app/moves/controllers/download.test.js

```diff
@@ -89,7 +92,7 @@ describe('Moves controllers', function() {
 
       it('should translate filename', function() {
         expect(req.t).to.be.calledOnceWithExactly('moves::download_filename', {
-          date: mockMoveDate,
+          date: '2018-10-10-2018-10-17',
```

Comment:
Wonder if it might be nicer to use `to` instead of `-`. Could be easier to see the split as currently there are a lot of `-`. Or if we want to keep 2, maybe we remove the dashes splitting the date parts - `20181010-20181017`?

Reply:
Let's use `to`